doc/flume_extensions_catalogue/grouping_interceptor.md

```diff
@@ -110,3 +111,40 @@ The usage of such an Interceptor is:
 It is <b>very important</b> to configure the <b>absolute path to the grouping rules file</b>.
 
 [Top](#top)
+
+##<a name="section4"></a>Management Interface related operations
+
+The Management Interface of Cygnus exposes a set of operations under the `/v1/groupingrules` path related to the groupiong rules feature, allowing listing/updating/removing the rules. For instance:
```

Comment:
Fixed in https://github.com/telefonicaid/fiware-cygnus/commit/ef5126c6d6c241b9b98e339a07d800efc58a88d6